pyproject.toml

```diff
@@ -36,12 +36,12 @@ dependencies = [
 dev = [
   "ariadne",
   "pytest",
-  "pylint",
+  "pylint==3.2.7",
   "mypy",
   "types-toml",
   "pytest-mock",
   "pytest-asyncio",
-  "pytest-httpx",
+  "pytest-httpx<=0.34.0",
```

Comment:
Pinned due to [requiring `httpx==0.28.0` since 0.35.0](https://github.com/Colin-b/pytest_httpx/blob/master/CHANGELOG.md#0350---2024-11-28)